estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,74 @@
+from dateutil.relativedelta import relativedelta
+from odoo.tools import float_compare
+
+from odoo import models, fields, api
```

Comment:
https://www.odoo.com/documentation/18.0/contributing/development/coding_guidelines.html#imports
same here, follow the guidelines , and update in all files
```suggestion
from odoo import api, fields, models
```

Reply:
Got it and rearranged